tests/verify_vector_table.py

```
@@ -0,0 +1,232 @@
+        )
+
+    if end <= start:
+        raise RuntimeError(
+            f"Vector table end {end:#06x} is not greater than start {start:#06x}."
+        )
+
+    if (end - start) % 4:
+        raise RuntimeError(
+            "Vector table size is not a multiple of 4 bytes: "
+            f"start={start:#06x}, end={end:#06x}"
+        )
```

Ruff (0.14.2)

Avoid specifying long messages outside the exception class

(TRY003)